tool/tsh/proxy.go

```diff
@@ -383,22 +412,44 @@ func onProxyCommandDB(cf *CLIConf) error {
 		if err != nil {
 			return trace.Wrap(err)
 		}
-		cmd, err := dbcmd.NewCmdBuilder(client, profile, routeToDatabase, rootCluster,
+		cmdMap, err := dbcmd.NewCmdBuilder(client, profile, routeToDatabase, rootCluster,
 			dbcmd.WithLocalProxy("localhost", addr.Port(0), ""),
 			dbcmd.WithNoTLS(),
 			dbcmd.WithLogger(log),
 			dbcmd.WithPrintFormat(),
-		).GetConnectCommand()
+		).GetConnectCommandOptions()
```

Comment:
I would rather keep this logic simple narrowed to one command for current usage.  There are hundreds of many elastic serach libraries like: python, go and if we want to support the multiple command templates it should be more generic. For instance this logic is shared with Teleport Connect so we will introduce a separate behaviour for teleport connect and this flow will diverge from Teleport Connect proxy flow. 
@Tener What would you say about narrowing it to interactive and none interactive modes where interactive will print `elasticserach-cli` and non interactive `curl` usage ?

Reply:
Sure, I've removed Python usage as requested.